typebackend/views.py

```diff
@@ -1,11 +1,18 @@
-from django.shortcuts import render
-from rest_framework.views import APIView
+from rest_framework.permissions import IsAuthenticated
 from rest_framework.renderers import JSONRenderer
-from rest_framework import status
 from rest_framework.response import Response
-from django.contrib.auth.models import User
 from .serializers import RegisterSerializer
-from rest_framework.permissions import IsAuthenticated
+from rest_framework.authtoken.models import Token
+from django.contrib.auth.models import User
+from google.auth.transport import requests
+from rest_framework.views import APIView
+from django.shortcuts import render
+from google.oauth2 import id_token
+from rest_framework import status
+from decouple import config
```

Comment:
Feel typebackend is growing in size. Feel it would be better to create a separate app called like "auth" to contain these oauth + user auth views.